src/components/pages/Work.jsx

```diff
@@ -148,6 +156,39 @@ const BadgeUrl = ({ workId, date }) => {
   )
 }
 
+const MakeClaimButton = () => (
+  <button className={classNames.makeClaimButton}>Make a claim</button>
+)
+
+const ContentTab = ({ work }) => (
+  <article>
+    <Content archiveUrl={work?.claim?.archiveUrl}/>
```

Comment:
Note for later: new claims have an `about` and it's an array. We'll still have older claims with `archiveUrl` and we'll need to support both. 

We'll want to do something like `<Content archiveUrl={getAboutX(work)}>` and define

```js
const getAboutX = work => isArray(work?.claim?.about) 
  ? selectNonPoetAboutUri(work.claim.about)
  : work?.claim?.archiveUrl

const selectNonPoetAboutUri = about => about.find(pipe(isPoetUri, not))

const isPoetUri = uri => uri.startsWith('poet:')
```

And also rename the prop `archiveUrl` to just `url`.